docs/usage.md

```diff
@@ -417,3 +417,75 @@ podman build . \
   --network none \
   --tag atomic-reactor
 ```
+
+### Example: npm
+
+Let's build simple npm project [sample-nodejs-app](https://github.com/cachito-testing/sample-nodejs-app).
+Get the repo if you want to try for yourself:
+
+
+```shell
+git clone https://github.com/cachito-testing/sample-nodejs-app.git
+```
+
+#### Pre-fetch dependencies (npm)
+
+The steps for pre-fetching the dependencies is similar to before, but this time we will use the `npm` package
+manager type. The default behavior path of `.` is assumed.
+
+See [the npm documentation](npm.md) for more details about running Cachi2 for pre-fetching npm dependencies.
+
+```shell
+cachi2 fetch-deps --source ./sample-nodejs-app --output /tmp/cachi2-output '{
+  "type": "npm",
+}'
+```
+
+#### Generate environment variables (npm)
+Currently, Cachi2 does not require to generate any environment variables for npm package manager.
+
+#### Inject project files (npm)
+
+In order to be able to install npm dependencies in a hermetic environment,
+we need to perform the injection to change the remote dependencies to instead point to the local file system.
+
+```shell
+cachi2 inject-files /tmp/cachi2-output --for-output-dir /tmp/cachi2-output
```

Comment:
if `{output_dir} == {for_output_dir}`, there's no need to use --for-output-dir

but the other examples use a different output dir

Reply:
OK I added different for_output_dir